src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -768,11 +768,13 @@ private void DynamoViewModelChangeScaleFactor(object sender, EventArgs e)
             var view = new Prompts.ChangeScaleFactorPrompt(dynamoViewModel.ScaleFactorLog) { Owner = this };
             if (view.ShowDialog() == true)
             {
-                if (dynamoViewModel.ScaleFactorLog != view.SliderValue)
+                if (dynamoViewModel.ScaleFactorLog != view.ScaleValue)
                 {
-                    dynamoViewModel.ScaleFactorLog = view.SliderValue;
+                    dynamoViewModel.ScaleFactorLog = view.ScaleValue;
                     dynamoViewModel.CurrentSpace.HasUnsavedChanges = true;
 
+                    Log("Scale factor log changed to " + view.ScaleValue);
```

Comment:
We need to use these formats with the commas for better readability and are trying to avoid scientific notation. Using the globalized `CultureInfo` doesn't seem to preserve the commas, converts 0.000001 to 1E-06, but yes, it does change 0.01 to 0,01 etc. which depending on user locale, can be useful. @Racel ?